src/main/java/seedu/address/model/person/Person.java

```diff
@@ -22,18 +22,19 @@
     private ObjectProperty<Phone> phone;
     private ObjectProperty<Email> email;
     private ObjectProperty<Address> address;
-
+    private ObjectProperty<Remark> remark;
     private ObjectProperty<UniqueTagList> tags;
 
     /**
-     * Every field must be present and not null.
+     * Constructor includes a remark field.
```

Comment:
Should change this back to the original comment, or improve on this comment. Currently it is missing the information from the previous comment

Reply:
oh i see,...at first I created two constructor....and finally i deleted one